### DISCUSSION TITLE
Doc typo

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#116] [Issue]
From https://keygen.sh/docs/api/authentication/:

> License tokens, also called license tokens, are specific to an individual license. 

Should perhaps be "License tokens, also called activation tokens...". This suggestion is based on the quote

> Authenticating with a license key is essentially the same as authenticating with [a license token](https://keygen.sh/docs/api/authentication/#license-tokens) (formerly known as an activation token).

The "formerly" seems little out-of-place since the interface still calls everything _activation token_:

![screenshot_2023-12-29-091434](https://github.com/keygen-sh/community/assets/181628/85b6a411-1bbf-4653-a025-f0815fcb1656)

**Comment by ezekg** [comment]
Thanks for the report. In the new upcoming Portal app, activation tokens have been renamed. Docs and current UI need to be updated to the new verbiage (and the botched find/replace in the docs needs to be fixed). Will do that soon.
